Services/WebContent/WebDriverConnection.cpp

```diff
@@ -251,43 +254,59 @@ void WebDriverConnection::set_is_webdriver_active(bool is_webdriver_active)
     current_browsing_context().page().set_is_webdriver_active(is_webdriver_active);
 }
 
+#define RESOLVE(...) resolve(Web::WebDriver::Response { __VA_ARGS__ })
+#define ER() resolve(Web::WebDriver::Response { JsonValue{} })
```

Comment:
this macro name "ER" is far too short to be useful

Reply:
Absolutely, it was just for peace of mind when renaming all this stuff. We have this convention scattered throughout `WebDriver` where we return (serialize and write back) empty json values, only to at some point later, ad-hoc respond with the actual value.

In the final product as it were, all macros will be removed (because all functions will resolve using their respective `Resolver`).